### PR TITLE
AGENT-165: Install release specified in ClusterImageSet

### PR DESCRIPTION
**Pull request by zaneb**
Instead of hard-coding the release image to be installed in the
assisted-service config, use the version passed in the ClusterImageSet
manifest. dev-scripts already provides this in test environments.

### CODE REVIEW

File: data/data/agent/files/usr/local/share/assisted-service/images.env.template [hunk 1]
```diff
@@ -1,3 +1,3 @@
...
+OS_IMAGES=[{"openshift_version":"4.10","cpu_architecture":"x86_64","url":"https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/4.10.0-rc.1/rhcos-4.10.0-rc.1-x86_64-live.x86_64.iso","rootfs_url":"https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/4.10.0-rc.1/rhcos-4.10.0-rc.1-x86_64-live-rootfs.x86_64.img","version":"410.84.202201251210-0"},{"openshift_version":"4.11","cpu_architecture":"x86_64","url":"https://rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com/art/storage/releases/rhcos-4.11/411.85.202203181601-0/x86_64/rhcos-411.85.202203181601-0-live.x86_64.iso","rootfs_url":"https://rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com/art/storage/releases/rhcos-4.11/411.85.202203181601-0/x86_64/rhcos-411.85.202203181601-0-live-rootfs.x86_64.img","version":"411.85.202203181601-0"}]
```
Comment (bfournie) [inline data/data/agent/files/usr/local/share/assisted-service/images.env.template:3]: Just wondering how this is used and why a new entry was added.

**Review by bfournie — COMMENTED** [review]
/lgtm

**Comment by bfournie** [comment]
Would be good to merge this after https://github.com/openshift/installer/pull/5942 and get ClusterImageSet as an asset.

File: data/data/agent/files/usr/local/share/assisted-service/images.env.template (hunk 1 above)
Comment (zaneb) [reply data/data/agent/files/usr/local/share/assisted-service/images.env.template:3]: https://github.com/openshift/installer/pull/5933/commits/11bdf17fff0cdae51ac4864aad1b927cdee92a3f

**Pull request merged**